### DISCUSSION TITLE
Not ignore #refs div for bibliography generated using bib

### DISCUSSION THREAD

**Discussion by jwang-lilly** [quarto-dev/quarto-cli#2703] [Q&A] [answered]
In the [quarto documentation]([](https://quarto.org/docs/authoring/footnotes-and-citations.html)), there is the following statement:

**If your bibliography is being generated using BibLaTeX or natbib ([Section 1.6](https://quarto.org/docs/authoring/footnotes-and-citations.html#sec-biblatex)), the bibliography will always appear at the end of the document and the #refs div will be ignored**

Is there any way to include output of sessionInfo() following References and Footnotes?

I'd like to have the following sequence:

1. References
2. Footnotes
3. SessionInfo() output
4. Copyright © 2022 My company. All rights reserved

Currently, the sequence is as follows:

1. SessionInfo() output
2. Copyright © 2022 My company. All rights reserved
3. References
4. Footnotes

**Comment by dragonstyle** [comment, marked as answer]
You could do this by modifying the underling template - that is controlling where the `\bibliography` is output (in `biblio.tex`). You can read more about how to approach this here:

https://quarto.org/docs/journals/templates.html#latex-partials

You could provide your own empty `biblio.tex` file (preventing Quarto/Pandoc from outputting any template information) and then place it all yourself, for example. 

Good luck!

**Reply by jwang-lilly** [reply]
thanks much @dragonstyle
